quasar/main.tf

```diff
@@ -280,7 +280,7 @@ resource "aws_db_parameter_group" "quasar-qa-pg11" {
   # Based on 50 connections.
   parameter {
     name  = "work_mem"
-    value = "20971"
+    value = "41943"
```

Comment:
Any ideas why this went up? I'd have thought it would be roughly cut in half.

Reply:
Yeah, I was curious about that as well, and re-ran the numbers on the current instance before this PR went in, and looks like we had a bad copy/paste from a configuration using double the `max_connections` we actually use. 😬

Reply:
Yikes! Good to catch it now.